docs/continuous-delivery/gitops/secret-management/secret-management-gitops.md

```diff
@@ -0,0 +1,177 @@
+- Does not scale with larger teams as each developer has to encrypt the secrets
+- The public key is sufficient for creating brand new files. The secret key is required for decrypting and editing existing files because SOPS computes a MAC on all values.  When using the public key solely to add or remove a field, the whole file should be deleted and recreated.
+
+## 2. Reference to secrets stored in an external key vault (recommended)
+
+This approach relies on a key management system like [Azure Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/general/overview) to hold the secrets and the git manifest in the repositories has reference to the key vault secrets. Developers do not perform any cryptographic operations with files in repositories. Kubernetes operators running in the target cluster are responsible for pulling the secrets from the key vault and making them available either as Kubernetes secrets or secrets volume mounted to the pod.
+
+For secret rotation ideas, see [Secrets Rotation on Environment Variables and Mounted Secrets](#secrets-rotation-on-environment-variables-and-mounted-secrets)
+
+For how to authenticate private container registries with a Service principal see: [Authenticated Private Container Registry](#authenticated-private-container-registry)
+
+All the below options provide the following:
+
+- Secrets are not stored in the repository
+- Supports Prometheus metrics for observability
+- Supports sync with Kubernetes Secrets
+- Supports Linux and Windows containers
+- Provides enterprise-grade external secret management
+- Easily scalable with multi-cluster and larger teams
+- Both solutions support either Service Principal or Managed Identity for [authentication with the Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/general/authentication).
+
+### [Azure Key Vault Provider for Secrets Store CSI Driver](https://github.com/Azure/secrets-store-csi-driver-provider-azure)
+
+Azure Key Vault Provider (AKVP) for Secrets Store CSI Driver is an azure vault provider for [Kubernetes secret store CSI Driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) allows you to get secret content stored in Azure Key vault instance and use the Secrets Store CSI driver interface to mount them into Kubernetes pods. Mounts secrets/keys/certs to pod using a CSI Inline volume.
+
+Azure Key Vault Provider for Secrets Store CSI Driver [install guide](https://secrets-store-csi-driver.sigs.k8s.io/getting-started/installation.html).
+
+CSI driver will need access to Azure Key vault either through  or managed identity(recommended). To make this access secure you can leverage [Azure Workload Identity](https://github.com/Azure/azure-workload-identity)(recommended) or [AAD Pod Identify](https://github.com/Azure/aad-pod-identity). Please note AAD pod identity will soon be replaced by Azure WorkLoad Identity.
```

Comment:
Missing word(s) - "either through <space> or managed identity"
Azure Workload Identity --> Azure AD Workload Identity (and in later one, the L in WorkLoad should be lower case)

Reply:
Tsk, tsk. You're losing your edge. You missed the misspelled `identify-> identity`